README.md

```diff
@@ -1,42 +1,56 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# todo-list
+# Project Name
+
+> Description the project.
 
-> A  project to keep track of to do list.
 
 ## Built With
 
-- HTML
-- CSS
-- Javascript
-- Webpack
+- Major languages
+- Frameworks
+- Technologies used
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
 
 ## Getting Started
 
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-- A browser
-- A version control like git
-- A text editor
-- npm
-
 ### Setup
 
-- Clone the repository using the link `https://github.com/bienvenuushindi/to-do-list.git`
-- Navigate to the folder using `cd to-do-list`
-- Run `npm install` to install all the dependencies
-- Run `npm start` to fire a development server
+### Install
+
+### Usage
```

Comment:
- [ ] Please it will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. Please add descriptive steps from cloning your app to seeing the results displayed on the browser. You can put the steps to clone and run the project in a new Getting Started section under the project's description. This includes `git clone URL` , `cd project_name`, `npm install`, `npm start`, etc 👨🏻‍🎨 🥇 
-This should be an example of how you may give instructions on setting up your project locally.